mautic/mautic
# [UX] Change "Has full system access?" label to "Is administrator?"

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR updates the user interface to enhance clarity around user permissions. By changing the label from “Has full system access?” to “Is administrator?”, we eliminate ambiguity and align with common terminology being used on user's settings. This change is accompanied by a tooltip that offers further details, ensuring users have a clear understanding of administrative roles within the system. The update benefits all users by providing a more intuitive experience and preventing potential misunderstandings regarding access levels.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Roles
3. Create new
4. Look at the label

![image](https://github.com/mautic/mautic/assets/116097999/7447588e-e868-4a0c-ad74-51bc4461d2a5)
